components/authentication/base/kustomization.yaml

```diff
@@ -1,20 +1,20 @@
 resources:
 - everyone-can-view.yaml
 - component-maintainer.yaml
-- spi.yaml
-- release-team.yaml
-- has.yaml
+# - spi.yaml
+# - release-team.yaml
+# - has.yaml
 - build.yaml
-- integration.yaml
-- cluster-registration-maintainer.yaml
+# - integration.yaml
+# - cluster-registration-maintainer.yaml
 - group-sync/
-- dora-metrics.yaml
-- internal-services.yaml
+# - dora-metrics.yaml
+# - internal-services.yaml
 - pipeline-service-sre.yaml
 - sandbox-sre-admins.yaml
-- has-admin.yaml
+# - has-admin.yaml
 - inspect-pods.yaml
-- segment-bridge.yaml
+# - segment-bridge.yaml
 
```

Comment:
let's remove the etries from here rather than commenting out.

Reply:
commented lines have been removed